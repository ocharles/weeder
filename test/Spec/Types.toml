roots = [ "Spec.Types.Usages.numberUsage"
        , "Spec.Types.Usages.modulo1Usage"
        , "Spec.Types.Usages.recordUsage"
        , "Spec.Types.Usages.vectorUsage"
        ]

type-class-roots = false

unused-types = true
