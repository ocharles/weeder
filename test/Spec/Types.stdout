main: test/Spec/Types/Usages.hs:16:1: notRoot
